blazor/dropdown-list/value-binding.md

```diff
@@ -220,6 +220,102 @@ Here, the DropDownList component is rendered with the TValue as a string type in
 {% endhighlight razor %}
 {% endtabs %}
 
+## Dynamically changing TItem using Templates
+
+Here, the DropDownList component is rendered with the TValue as a string type in the `/Index.razor` file and the DropDownList component with TValue as an int nullable type in the `/Counter.razor` file using [ItemTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownBase-1.html#Syncfusion_Blazor_DropDowns_SfDropDownBase_1_ItemTemplate) and [ValueTemplate](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_ValueTemplate).
+
+**[Index.razor]**
+
+{% tabs %}
+{% highlight razor %}
+
+<DropDownList TVal="string" TItemss="Games" @bind-DDLValue="@value" customData="@LocalData">
+    <itemTemplate Context="data">
+        <div>@data.Text</div>
+    </itemTemplate>
+    <valueTemplate Context="model">
+        <div>@model.Text - @model.ID</div>
+    </valueTemplate>
+</DropDownList>
+
+@code{
+    public string value { get; set; }
+    public class Games
+    {
+        public string ID { get; set; }
+        public string Text { get; set; }
+    }
+    List<Games> LocalData = new List<Games> {
+new Games() { ID= "Game1", Text= "American Football" },
+new Games() { ID= "Game2", Text= "Badminton" },
+new Games() { ID= "Game3", Text= "Basketball" },
+new Games() { ID= "Game4", Text= "Cricket" },
+new Games() { ID= "Game5", Text= "Football" },
+new Games() { ID= "Game6", Text= "Golf" },
+new Games() { ID= "Game7", Text= "Hockey" },
+new Games() { ID= "Game8", Text= "Rugby"},
+new Games() { ID= "Game9", Text= "Snooker" },
+new Games() { ID= "Game10", Text= "Tennis"},
+  };
+}
+
+{% endhighlight razor %}
+{% endtabs %}
+
+**[Counter.razor]**
+
+{% tabs %}
+{% highlight razor %}
+@using Syncfusion.Blazor.DropDowns;
+@typeparam TVal;
+@typeparam TItemss;
+
+<SfDropDownList TValue="TVal" TItem="TItemss" @bind-Value="@DDLValue" DataSource="@customData" ItemTemplate="@itemTemplate" ValueTemplate="@valueTemplate">
+    <DropDownListFieldSettings Text="Text" Value="ID"></DropDownListFieldSettings>
+</SfDropDownList>
+
+@code {
+
+    [Parameter]
+    public List<TItemss> customData { get; set; }
+    [Parameter]
+
+    public TVal DDLValue { get; set; }
+    [Parameter]
+    public EventCallback<TVal> DDLValueChanged { get; set; }
+    [Parameter]
+    public RenderFragment<TItemss> itemTemplate { get; set; }
+    [Parameter]
+    public RenderFragment<TItemss> valueTemplate { get; set; }
+    }
+
+{% endhighlight razor %}
+{% endtabs %}
+
+## Customizing TItem using action keyword
+
+In the following code snippet, action keyword is using in the `TItem` of the dropdown list.

```

Comment:
In the following code sample, the action keyword is used in the `TItem` of the dropdown list.